app/api/v2/handlers/operation_api.py

```diff
@@ -227,7 +227,7 @@ async def get_operation_link(self, request: web.Request):
                           }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
     @aiohttp_apispec.response_schema(LinkSchema(partial=True),
-                                     description='Contains a result string for the link requested.')
+                                     description='Contains a result dictionary as a string for the link requested.')
```

Comment:
```suggestion
                                     description='Contains a dictionary of the requested link object and link's result.')
```

Reply:
Added a little clarification to this one, but feel free to tweak as you see fit. You could also add in the note about the base64 encoding if you feel it's appropriate.